### DISCUSSION TITLE
表示可以把那个绿色的提示条加回来吗？

### DISCUSSION THREAD

**Discussion by Malus-Mill** [andywang425/BLTH#272] [General]
如题
表示没有提示条感觉就像没运行一样

**Comment by andywang425** [comment]
之前某个版本减少了提示，是因为提示太多了看不清，等于没提示。所以大部分任务只在开始和结束的时候提示。  
如果是普通用户的话没必要去关注运行细节，比较详细的日志会显示在控制台。

**Reply by Malus-Mill** [reply]
表示控制台开了没有显示

**Reply by Malus-Mill** [reply]
表示有5、600左右的牌子、加了提示有助于查看进度

**Reply by andywang425** [reply]
> 表示控制台开了没有显示

显示的大多是一些API的response，可能不是很容易看懂。

> 表示有5、600左右的牌子、加了提示有助于查看进度

显示日志和提示总得有取舍，现在脚本功能太多，开启的功能的一多日志很容易刷屏，不必要的日志和提示还是得少加。

**Reply by Malus-Mill** [reply]
表示开了日志、感觉良好
